### DISCUSSION TITLE
Configs reset on Android?

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#512] [Q&A] [answered]
This is a rather general inquiry, maybe someone has run into this before or maybe there is already a fix that I haven't located.

During a session on my phone I check the game configuration to make sure that e.g. wolves don't spawn indefinitely.
I save the game and navigate to another app on my phone. Sometimes I quit Augustus (remove it from the list of running apps), sometimes I don't.
When I navigate back to the app, I occasionally get directly to the city, when the app was still running in background, so I don't need to load the game, like when starting Augustus fresh.
In most of those cases (maybe all, I haven't kept track) I end up having to hunt down wolves again - sometimes after 2 game months, sometimes after more than a year. And checking the config I find that all the settings have been reset.

Has anyone else faced this issue?

**Comment by crudelios** [comment, marked as answer]
This is a known 3.0.1 issue that is fixed on the latest unstable, so either you should get that version or wait for v3.1.0 which shouldn't be too far away.

**Reply by catalin-duta** [reply]
I'm on 3.0.1.280 (unstable build from a few days back), I'll try out .281.
Just so I don't ask any more questions on what's likely already documented: would I find this information anywhere?
I did not see it under any of the tabs at the top...

**Reply by crudelios** [reply]
The only documentation is the commit information. Just to be sure, are you having this issue with the unstable build as well?

**Reply by catalin-duta** [reply]
I had this at least with unstable build 3.0.1.280. Will see with 3.0.1.281.

**Reply by crudelios** [reply]
No need, it isn't fixed yet, though it should have been fixed when we merged julius 1.7.0. I'll have to check this.

**Reply by crudelios** [reply]
It should be fixed now, try it when 3.0.1.282 is ready for download.
